### PR TITLE
EN-2337 DE PageModels

### PR DESCRIPTION
**Pull request by zonia3000**
As asked by FE I added also an `installed` flag on the DE PageModels response and I explicitly pointed out the usage of the exchange id or name (using always `digitalExchangeName` and `digitalExchangeId` instead of `digitalExchange`).

### CODE REVIEW

File: plugins/entando-plugin-jpdigital-exchange/src/main/java/org/entando/entando/aps/system/services/pagemodel/DigitalExchangePageModelListProcessor.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                case MAIN_FRAME:
+                    return p -> FilterUtils.filterInt(filter, p::getMainFrame);
+                case PLUGIN_CODE:
+                    return p -> FilterUtils.filterString(filter, p::getPluginCode);
+                case TEMPLATE:
+                    return p -> FilterUtils.filterString(filter, p::getTemplate);
+                case DIGITAL_EXCHANGE:
+                    return p -> FilterUtils.filterString(filter, p::getDigitalExchange);
+            }
+            return null;
```
Comment (rzsouza) [inline plugins/entando-plugin-jpdigital-exchange/src/main/java/org/entando/entando/aps/system/services/pagemodel/DigitalExchangePageModelListProcessor.java]: A possible improvement is to move this return null to a default case

File: plugins/entando-plugin-jpdigital-exchange/src/main/java/org/entando/entando/aps/system/services/pagemodel/DigitalExchangePageModelListProcessor.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                    return (a, b) -> a.getPluginCode().compareTo(b.getPluginCode());
+                case MAIN_FRAME:
+                    return (a, b) -> Integer.compare(a.getMainFrame(), b.getMainFrame());
+                case PLUGIN_CODE:
+                    return (a, b) -> a.getPluginCode().compareTo(b.getPluginCode());
+                case TEMPLATE:
+                    return (a, b) -> a.getTemplate().compareTo(b.getTemplate());
+                case DIGITAL_EXCHANGE:
+                    return (a, b) -> a.getDigitalExchange().compareTo(b.getDigitalExchange());
+                case CODE:
```
Comment (rzsouza) [inline plugins/entando-plugin-jpdigital-exchange/src/main/java/org/entando/entando/aps/system/services/pagemodel/DigitalExchangePageModelListProcessor.java]: Maybe a comment here saying that the fall through is intentional

File: plugins/entando-plugin-jpdigital-exchange/src/main/java/org/entando/entando/aps/system/services/pagemodel/DigitalExchangePageModelListProcessor.java (hunk 1 above)
Comment (zonia3000) [reply plugins/entando-plugin-jpdigital-exchange/src/main/java/org/entando/entando/aps/system/services/pagemodel/DigitalExchangePageModelListProcessor.java]: done

File: plugins/entando-plugin-jpdigital-exchange/src/main/java/org/entando/entando/aps/system/services/pagemodel/DigitalExchangePageModelListProcessor.java (hunk 2 above)
Comment (zonia3000) [reply plugins/entando-plugin-jpdigital-exchange/src/main/java/org/entando/entando/aps/system/services/pagemodel/DigitalExchangePageModelListProcessor.java]: done

**Comment by rzsouza** [comment]
retest this please

**Comment by entando-service** [comment]
All is Well

**Pull request merged**